### ISSUE TITLE
[A11y] General -Page Title

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1807]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

Other than the homepage, all the page titles have "//" in them.  This is read out by screen readers by default, unlike hyphenated page titles, or the pipe character ("|").

I do not believe this is a hard failure, and it may even be preferable to some users; I don't know.  But it's definitely not as common as a pipe or a dash.  Note that if a dash is used, it should be the "em dash", not a hyphen: `—`.

### Suggested Fix

Uncertain if this even needs a fix

**Comment by CGillen** [comment]
Fixed by #1841
